api/graphql/schema.graphql

```diff
@@ -1568,6 +1577,23 @@ input CreatePoolAdvertisementInput {
   classifications: ClassificationBelongsToMany
 }
 
+input CreateScreeningQuestionInput {
+  question: LocalizedStringInput!
+  sortOrder: Int @rename(attribute: "sort_order")
+}
+
+input UpdateScreeningQuestionInput {
+  id: ID!
+  question: LocalizedStringInput!
```

Comment:
For updates, this should be nullable in case you don't want to change the questions.

Reply:
yeah, that is true https://github.com/GCTC-NTGC/gc-digital-talent/pull/6089/commits/e510bc5c24416f555f6a709f1b1a093c249e7818